### PR TITLE
Wrote branch-specific README

### PR DESCRIPTION
**Pull request by ojustino**
If there is more that should be added, please let me know.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+About
+-----
+
+This branch of pyWWT hosts data files for sky surveys. At the moment, we only
```
Comment (astrofrog) [inline README.rst]: Maybe say 'footprints'? Note that this branch also stored the JS files required by PyWWT, so it might be worth mentioning.

Comment (ojustino) [reply README.rst]: "Footprints" in place of "data files"?

Comment (astrofrog) [reply README.rst]: Maybe it should say 'This branch of the PyWWT repository hosts data files needed by the pywwt package, including instrument footprints as well as javascript files'.

**Comment by ojustino** [comment]
OK, I made a revision in the latest commit.

**Pull request merged**